src/page-view.ts

```diff
@@ -24,15 +25,21 @@ interface ScrollProperties {
     $prev_pageview_max_content_percentage?: number
 }
 
+interface PageViewEventProperties extends ScrollProperties {
+    $prev_pageview_pathname?: string
+}
+
 export class PageViewManager {
     _pageViewData: PageViewData | undefined
     _hasSeenPageView = false
 
     _createPageViewData(): PageViewData {
-        return {}
+        return {
+            pathname: utils.window.location.pathname,
```

Comment:
I assume we need this to query on? Do we perhaps want to just have a full url which allows any kind of grouping filtering later on?